clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/converters/RebootInstancesAtomicOperationConverter.groovy

```diff
@@ -47,7 +47,7 @@ class RebootInstancesAtomicOperationConverter extends AbstractAtomicOperationsCr
     try {
       def serverGroups = converted.instanceIds.findResults {
         def instance = amazonInstanceProvider.getInstance(converted.credentials.name, converted.region, it)
-        return instance?.extraAttributes?.serverGroup
```

Comment:
in groovy, `extraAttributes` is equivalent to calling the getter method `getExtraAttributes`